py3status/modules/mpd_status.py

```diff
@@ -91,24 +101,48 @@ class Py3status:
     # available configuration parameters
     cache_timeout = 2
     format = '{state} [[[{artist}] - {title}]|[{file}]]'
-    hide_when_paused = False
-    hide_when_stopped = True
+    hide_when_pause = False
+    hide_when_play = False
+    hide_when_stop = True
     host = 'localhost'
     max_width = 120
+    mpd_no_auth = 'mpd: authentication failed'
+    mpd_no_conn = 'mpd: connection refused'
```

Comment:
I'm thinking `mpd_connection` and `mpd_authentication`? I guess we could do `state_connection` and `state_authentication` instead. Thoughts? 

I don't care for `mpd_no_auth` myself as I don't use passworded mpd, but should leave for others. We especially want to keep `state_connection` in so we don't get that noise if we're not running mpd anywhere (similar to bluetooth, but not exception).